### DISCUSSION TITLE
Issue installing dependencies

### DISCUSSION THREAD

**Discussion by dguisti** [jertel/vuegraf#22] [Q&A] [answered]
Hello! I have a couple questions about installing the package. First, do I need to separately install InfluxDB, and if so, is that possible to do on windows? Secondly, when I try to install the pyemvue PyPI package pip errors out in the cryptodome setup.py file. Here is a traceback:
```
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.27.29110\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -DLTC_NO_ASM -Isrc/ -IC:\Users\me\AppData\Local\Programs\Python\Python39\include -IC:\Users\me\AppData\Local\Programs\Python\Python39\include -IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.27.29110\ATLMFC\include -IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.27.29110\include /Tcsrc/MD2.c /Fobuild\temp.win-amd64-3.9\Release\src/MD2.obj
    MD2.c
    C:\Users\me\AppData\Local\Temp\pip-install-4mia2j5q\pycryptodome_041d4291706e43c7963290d2af954d12\src\pycrypto_common.h(43): fatal error C1083: Cannot open include file: 'malloc.h': No such file or directory
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.27.29110\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2
 ```
 
 Any help would be appreciated!

**Comment by jertel** [comment, marked as answer]
Hello, unfortunately i don't have an answer to your question about running python on windows. 

I suggest installing Docker on Windows instead. This allows you to avoid having to build anything. It also will allow you to run influx db and Grafana docker containers on Windows.

**Reply by dguisti** [reply]
Alright. I am quite new to docker, so I was curious if you could point me in the right direction. Would I build influx and grafana and vuegraf all in different containers? And do I need to download and run the python script still in order for the vuegraf docker container to work? I noticed that in your sample docker container launch command it included references to vuegraf config files in various locations. Do I need to put those there myself?

Thanks again for your help,
@dguis

**Reply by jertel** [reply]
I added some additional instructions to the README.md that should help point you in the right direction. The vuegraf.json file needs to be created by you, using the example as a starting point, and placed inside of a vuegraf folder/directory that you have created somewhere. Then you will adjust the Docker `run` command to use that referenced location (the portion on the left side of the colon : character).

**Reply by dguisti** [reply]
Alright. I got the grafana and influxDB docker containers working. However, I installed influxDB 2.0 and it seems to use a token system instead of credentials like is sown in the config files. Is it possible to use the vuegraf code with influxDB 2, or should I try to install an older version?

EDIT: It looks like the influx API for version 1.x is outdated for version 2 and a lot of the python interfaces have changed. I can go through and modify the python file appropriately, but I'm not sure how to update those changes to the docker container.

**Reply by jertel** [reply]
Use the 1.8 version for now. There will need to be changes to support the 2.x version.

**Reply by dguisti** [reply]
Alright
